source/content/guides/composer-convert.md

```diff
@@ -150,7 +183,19 @@ Follow suit with any other custom code you need to carry over.
 
 Your existing site may have customizations to `settings.php` or any other config files. Review these carefully and extract relevant changes from these files to copy over. Always review any file paths referenced in the code, as these paths may change in the transition to Composer.
 
-It is not wise to completely overwrite the  `settings.php` file with the old one, as there are customizations for moving the configuration directory you don't want to overwrite, as well as platform specific customizations.
+It is not wise to completely overwrite the `settings.php` file with the old one, as there are customizations for moving the configuration directory you don't want to overwrite, as well as platform specific customizations.
+
+```bash{promptUser:user}
+# Ensure working tree is clean
+git status
+git checkout master
+git checkout -b old-settings-php
+git mv sites/default/settings.php web/sites/default
+git commit -m 'Relocate settings.php'
+git checkout composerify
+# Interactively extract relevant hunks from old settings.php
+git checkout -p old-settings-php web/sites/default/settings.php
+```
```

Comment:
It's kind of a lot of steps just to get your settings.php in the right place on a separate branch (you also need a `mkdir -p web/sites/default` in the instructions above, as `web` does not exist on the original branch).

How about this instead:
```
git checkout master sites/default/settings.php
diff -Nup web/sites/default/settings.php sites/default/settings.php 
# Edit settings.php as needed
rm sites/default/settings.php
```
You don't get to use `git checkout -p`, but there are significantly fewer steps.

Reply:
can we resolve this conversation?

Reply:
Actually we need one more command before the `rm`:
```
git restore --staged sites/default/settings.php 
```
I don't know if there's a better way to do this.

Reply:
OK, here's a good solution:
```
git show master:sites/default/settings.php > web/sites/default/original-settings.php
diff -Nup --ignore-all-space web/sites/default/settings.php web/sites/default/original-settings.php
# edit web/sites/default/settings.php and commit as needed
rm web/sites/default/original-settings.php
```

Reply:
updated to match your latest comment, but I've kept the `git status` to catch people who're just running through steps and might have missed something staged
I can remove it if it's too in the way

Reply:
`git status` is a good idea.